pilot/pkg/serviceregistry/kube/controller/ambient/ambientindex.go

```diff
@@ -255,10 +255,6 @@ func (a *index) Lookup(key string) []model.AddressInfo {
 
 	// 3. Service
 	if svc := a.lookupService(key); svc != nil {
-		vips := sets.New[string]()
```

Comment:
Curious: what was this attempting to accomplish? Validation?

Reply:
Probably deadcode in a refactor is my guess.